src/50_Samples_%26_Templates/Checkout_Flow.html

```diff
@@ -0,0 +1,416 @@
+<!---{
+  "preview": "default",
+  "default": "preview"
+  }--->
+  <!--
+  ## Introduction
+
+  This sample demonstrates how you can implement a simple checkout page in AMP. The sample assumes that all payment processing is done server-side. The covered use cases are:
+
+  * How to dynamically render shopping card data.
+  * How to support user login with stored addresses and credit cards. 
+  * How to let users auto-fill their contact, address and credit card details.
+  * How to handle promo/discount codes. 
+  -->
+  <!-- -->
+  <!doctype html>
+  <html ⚡>
+    <head>
+      <meta charset="utf-8">
+      <link rel="canonical" href="<%host%>/../samples_templates/checkout_flow/">
+      <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
+      <script async src="https://cdn.ampproject.org/v0.js"></script>
+
+      <script async custom-template="amp-mustache" src="https://cdn.ampproject.org/v0/amp-mustache-0.1.js"></script>
```

Comment:
In previous samples we added a line explaining each import, do you want to follow this convention?

Reply:
yes, you're right